src/webview/add-service-binding/app/addServiceBindingForm.tsx

```diff
@@ -0,0 +1,201 @@
+                                </Select>
+                                <FormHelperText>
+                                    The Operator-backed service to connect to your component
+                                </FormHelperText>
+                            </FormControl>
+                            <FormControl>
+                                <TextField
+                                    id="binding-name"
+                                    variant="outlined"
+                                    label="Binding Name"
+                                    onClick={(_e) => {
+                                        if (!bindingNameTouched) {
+                                            setBindingNameTouched(true);
+                                        }
+                                    }}
+                                    onChange={(e) => {
+                                        setBindingName(e.target.value);
+                                    }}
+                                    error={bindingNameTouched && !isBindingNameValid(bindingName)}
+                                    required
+                                ></TextField>
+                                <FormHelperText>
+                                    The name of the <code>ServiceBinding</code> Kubernetes resource.
+                                    Can only contain letters, numbers, and dashes (<code>-</code>).
+                                </FormHelperText>
+                            </FormControl>
+                            <Grid container>
+                                <Grid item xs="auto">
+                                    {/* Instead of disabling the button when the form entries are invalid,
+                                    completely remove it, and instead show an alert explaining what needs to be fixed */}
+                                    {!isBindingNameValid(bindingName) || selectedService === '' ? (
+                                        <Alert severity="error">
+                                            You must&nbsp;
+                                            {selectedService === '' && (
+                                                <>select a service to bind to</>
+                                            )}
+                                            {!isBindingNameValid(bindingName) &&
+                                                selectedService === '' && <> and </>}
+                                            {!isBindingNameValid(bindingName) && (
+                                                <>set a valid binding name</>
+                                            )}
+                                            .
+                                        </Alert>
+                                    ) : (
+                                        <Button variant="contained" size="large" type="submit">
+                                            Add Service Binding
```

Comment:
This ends up showing up as all capitalized, maybe change the style to be consistent with the other webviews?